src/backend/cpu/CMakeLists.txt

```diff
@@ -47,11 +47,24 @@ IF(NOT UNIX)
     ADD_DEFINITIONS(-DAFDLL)
 ENDIF()
 
+INCLUDE(ExternalProject)
```

Comment:
Create a build_threads.cmake file in CMakeModules for this external project.
Follow clBLAS.